language/ir-testsuite/tests/signer/address_arg_is_not_signer.mvir

```diff
@@ -0,0 +1,39 @@
+module M {
+    resource R { s: signer }
+    public store_signer(s: signer) {
+        move_to_sender<R>(R { s: move(s) });
+        return;
+    }
+}
+
+//! new-transaction
+//! args: 0x0
```

Comment:
I believe `INVALID_FUNCTION_SIGNATURE` gets reported before binding params/args, so the `//!  args` here and in other tests that expect `INVALID_FUNCTION_SIGNATURE` can be removed.

Reply:
It does... sort of. The thing is that I'm doing a lot of magic under the hood to introduce the `&signer`

We want tests that make sure that changes in other areas don't affect core things like `INVALID_FUNCTION_SIGNATURE`. As such, both tests are testing different things, and should be kept